charts/scalardl-ledger-monitoring/values.yaml

```diff
@@ -607,91 +508,4 @@ alloy:
     configMap:
       content: |
-        discovery.kubernetes "scalardl_ledger" {
-          role = "pod"
-          namespaces {
-            names = ["{{ .Release.Namespace }}"]
-          }
-          selectors {
-            role = "pod"
-            label = "app.kubernetes.io/app=ledger"
-          }
-        }
-
-        discovery.kubernetes "envoy" {
-          role = "pod"
-          namespaces {
-            names = ["{{ .Release.Namespace }}"]
-          }
-          selectors {
-            role = "pod"
-            label = "app.kubernetes.io/app=envoy"
-          }
-        }
-
-        discovery.kubernetes "scalar_admin_for_kubernetes" {
-          role = "pod"
-          namespaces {
-            names = ["{{ .Release.Namespace }}"]
-          }
-          selectors {
-            role = "pod"
-            label = "app.kubernetes.io/app=scalar-admin-for-kubernetes"
-          }
-        }
-
-        discovery.relabel "replace_label_to_pod_scalardl_ledger" {
-          targets = discovery.kubernetes.scalardl_ledger.targets
-          rule {
-            source_labels = ["__meta_kubernetes_pod_name"]
-            action        = "replace"
-            target_label  = "pod"
-          }
-        }
-
-        discovery.relabel "replace_label_to_pod_envoy" {
-          targets = discovery.kubernetes.envoy.targets
-          rule {
-            source_labels = ["__meta_kubernetes_pod_name"]
-            action        = "replace"
-            target_label  = "pod"
-          }
-        }
-
-        discovery.relabel "replace_label_to_pod_scalar_admin_for_kubernetes" {
-          targets = discovery.kubernetes.scalar_admin_for_kubernetes.targets
-          rule {
-            source_labels = ["__meta_kubernetes_pod_name"]
-            action        = "replace"
-            target_label  = "pod"
-          }
-        }
-
-        loki.source.kubernetes "scalardl_ledger" {
-          targets    = discovery.relabel.replace_label_to_pod_scalardl_ledger.output
-          forward_to = [loki.relabel.remove_unnecessary_labels.receiver]
-        }
-
-        loki.source.kubernetes "envoy" {
-          targets    = discovery.relabel.replace_label_to_pod_envoy.output
-          forward_to = [loki.relabel.remove_unnecessary_labels.receiver]
-        }
-
-        loki.source.kubernetes "scalar_admin_for_kubernetes" {
-          targets    = discovery.relabel.replace_label_to_pod_scalar_admin_for_kubernetes.output
-          forward_to = [loki.relabel.remove_unnecessary_labels.receiver]
-        }
-
-        loki.relabel "remove_unnecessary_labels" {
-          forward_to = [loki.write.loki.receiver]
-          rule {
-            action   = "labelkeep"
-            regex    = "pod"
-          }
-        }
-
-        loki.write "loki" {
-          endpoint {
-            url = "http://{{ .Release.Name }}-loki:3100/loki/api/v1/push"
-          }
-        }
+        {{ include "scalardl-ledger-monitoring.alloyContent" . }}
```

Comment:
The Alloy's content configuration for Envoy is set here, `alloy.configMap.content`, if we set `global.envoy.enabled=true`.